### PR TITLE
Added Chatrix to clients page

### PR DESCRIPTION
**Pull request by navyansh007**
Added relevant client information for Chatrix on the Clients page of the website. This includes the markdown file containing information about Chatrix and the SVG file for its logo.

### CODE REVIEW

**Comment by HarHarLinks** [comment]
resolves #2213

**Comment by thibaultamartin** [comment]
@akirk I see the last commit was 2 months ago, is Chatrix maintained :) ?

**Comment by akirk** [comment]
Well, we'll fix bugs when needed and release updates, but at the moment the features are enough for WordPress.org, so we're not adding any new features for now.

**Review by thibaultamartin — CHANGES_REQUESTED** [review]

File: content/ecosystem/clients/chatrix.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+featured = true
+[extra.features]
+e2ee = ""
+spaces = ""
+voip_1to1 = "" 
+voip_jitsi = ""
+threads = ""
+sso = ""
+multi_account = ""
+multi_language = ""
```
[the remark this answers is not in the document]
Comment (navyansh007) [reply content/ecosystem/clients/chatrix.md]: Hello there, I have made the requested changes in my branch's code. I request you to please have a look at it.

**Comment by akirk** [comment]
As a bit of evidence :-) we just [released 0.9.1 of Chatrix](https://github.com/Automattic/chatrix/releases/tag/0.9.1).

**Comment by navyansh007** [comment]
I have modified the latest release date as well for Chatrix. I request you to please have a look at it.

**Comment by navyansh007** [comment]
Hello there @thibaultamartin, it's been around two weeks since I pushed my last changes in the PR. May you please let me know if there is any issue with my PR, that I need to resolve before it can be merged?

**Review by MTRNord — APPROVED** [review]

**Pull request merged**